index.html

```diff
@@ -0,0 +1,146 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale =1.0">
+        <title>Portfolio project</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/d901e0cb9a.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+      
+        <header>
+          <nav>
+            <ul>
+              <li><a class="logolink" href=""><b>My Logo</b></a></li>
+              <li style="float:right;" class="dropdown">
+<a href=""><img src="hammenu.png" alt="ham" style="width: 90px;height: 60px;"></a></li>
+             </ul>
+                    </nav>
+          <div class="textdiv"><h1>I’m Alexandra <br>
```

Comment:
- [ ] The `header` should be closed here right after the nav's closing tag. The headline section should than begin right after the header's closing tag, and just like the previous reviewer requested, it should be wrapped in a `section` tag. So after the header's closing tag you should have, `<section id="">`. Please make these changes.

Reply:
@NduatiKagiri Actually I am using a background image taken from FIGMA design layout and using that image as a background image for the header. And the Heading Text is appearing on the background image that is why I defined Headings Tags inside the header that way I can position my Heading over the Background image of the header area. 
Yes it was mentioned by previous Code Reviewer to include my Icons in a section that I did in a section. Please allow me to include the the heading text inside the header otherwise it would become difficult for me to position my heading over the background image.

Reply:
And secondly yes I am working on making suggested changes by you for font styles and font sizes  to include Poppins font from Google Fonts. Those changes, I will incorporate within the Project and will resend for your kind review again.
Thanks and regards,
Baqar Abbas